src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+	protected String description;
+	protected Boolean isDone;
+
+	public Task(String description) {
+		this.description = description;
+		this.isDone = false;
+	}
+
+	public String getStatusIcon() {
+		return (isDone ? "[" + "\u2713" + "]" : "[" + "\u2718" + "]"); //returns [tick] or [X] symbols
+	}
+
+	public void markAsDone() {
+		this.isDone = true;
+	}
+
+	@Override
+	public String toString() {
+		return getStatusIcon() + " " + this.description;
+	}
```

Comment:
The indentation seems a bit different from the Duke.java file. Did you use 8 spaces instead of 4 for this file?